### PR TITLE
GRW-2217 - Update AccordionBlock so it uses grid layout system

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Set AccordionBlock spacing according to design.

## Jira issue(s): [GRW-2217](https://hedvig.atlassian.net/browse/GRW-2217)

[GRW-2217]: https://hedvig.atlassian.net/browse/GRW-2217?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1606** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1606" target="_blank">Graphite</a>  👈
    * **PR #1607** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1607" target="_blank">Graphite</a> 
      * **PR #1608** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1608" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1606?utm_source=stack-comment).

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 1]
```diff
@@ -54,26 +55,27 @@ export const AccordionBlock = ({ blok }: Props) => {
 }
 AccordionBlock.blockName = 'accordion'
 
-const Wrapper = styled.div({
-  display: 'flex',
-  flexWrap: 'wrap',
-  gap: theme.space.lg,
+const Wrapper = styled(GridLayout.Root)({
```
Comment (robinandeer) [inline apps/store/src/blocks/AccordionBlock.tsx:58]: Nice stuff! 

Let's collect all the cases in the future and review how we can enable them from the same base layout component

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 2]
```diff
@@ -54,26 +55,27 @@ export const AccordionBlock = ({ blok }: Props) => {
...
+  },
 })
 
 const TitleDescriptionWrapper = styled.div({
-  flex: 1,
-  minWidth: 'min(23.75rem, 100%)',
+  gridColumn: '1 / -1',
+  [mq.lg]: {
+    gridColumn: 'span 6',
+    maxWidth: '37.5rem',
```
Comment (robinandeer) [inline apps/store/src/blocks/AccordionBlock.tsx:69]: is there a relevant variable name / comment for this value?

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 3]
```diff
@@ -54,26 +55,27 @@ export const AccordionBlock = ({ blok }: Props) => {
...
-  minWidth: 'min(28.125rem, 100%)',
-
-  [mq.md]: {
-    gap: theme.space.xs,
+  gridColumn: '1 / -1',
+  margin: `${theme.space.lg} 0 0 0`,
+  [mq.lg]: {
+    gridColumn: 'span 6',
+    margin: `0 0 0 ${theme.space.xxl}`,
   },
```
Comment (robinandeer) [inline apps/store/src/blocks/AccordionBlock.tsx:79]: can we change to just `marginTop` and `marginLeft` -- I can never remember this syntax 😅

**Review by robinandeer — APPROVED** [review]
Nice great stuff

**Pull request merged**